src/action/common/configure_init_service.rs

```diff
@@ -157,23 +161,38 @@ impl Action for ConfigureInitService {
                 )
                 .await?;
 
-                execute_command(
-                    Command::new("systemctl")
-                        .process_group(0)
-                        .arg("link")
-                        .arg(SERVICE_SRC)
-                        .stdin(std::process::Stdio::null()),
-                )
-                .await?;
+                if Path::new(SERVICE_DEST).exists() {
+                    // TODO: ask user if we can remove it...? do we have a way
+                    // to model interaction within the plan?
+                    // TODO: use a more appropriate error like RemoveExisting or something
+                    tokio::fs::remove_file(SERVICE_DEST)
+                        .await
+                        .map_err(|e| ActionError::Remove(PathBuf::from(SERVICE_DEST), e))?;
+                }
+                tokio::fs::symlink(SERVICE_SRC, SERVICE_DEST)
```

Comment:
Switched to `symlink` directly because we know where it is and where it should be.

Reply:
I am fairly sure we couldn't do this with the steam deck? Might be worth checking.

Reply:
I validated and this works just fine, it seems.

Installed, ran `nix run nixpkgs#hello | nix run nixpkgs#ponysay`, rebooted, ran `nix run nixpkgs#hello | nix run nixpkgs#ponysay` again, worked fine.

Reply:
Interesting, for some reason I thought this would give issues with cross mount symlinks...

Reply:
AFAIK, this is just a simplification of what `systemctl link` does anyways:

https://github.com/systemd/systemd-stable/blob/8d1db692194f7db407754e85961d82d99117ddab/src/systemctl/systemctl-enable.c#L119 -> https://github.com/systemd/systemd-stable/blob/8d1db692194f7db407754e85961d82d99117ddab/src/shared/install.c#L2407 -> https://github.com/systemd/systemd-stable/blob/8d1db692194f7db407754e85961d82d99117ddab/src/shared/install.c#L2488 -> https://github.com/systemd/systemd-stable/blob/8d1db692194f7db407754e85961d82d99117ddab/src/shared/install.c#L559